services/headless-lms/migrations/20240527014445_add_course_instance_id_to_suspected_cheaters_and_cheater_thresholds.up.sql

```diff
@@ -15,3 +15,6 @@ ADD CONSTRAINT unique_course_id_constraint UNIQUE USING INDEX unique_course_id;
 ALTER TABLE course_module_completions -- this is not breaking the db because of the default
 ADD COLUMN needs_to_be_reviewed BOOLEAN DEFAULT NULL;
 COMMENT ON COLUMN course_module_completions.needs_to_be_reviewed IS 'Determine if a course module needs review as a result of a student being suspected of cheating';
+--Add is_archived to suspected cheater table
+ALTER TABLE suspected_cheaters
```

Comment:
Add a comment to the column explaining what all the three states are for (null, true, false)